arches/app/datatypes/datatypes.py

```diff
@@ -320,10 +321,14 @@ def transform_value_for_tile(self, value, **kwargs):
             value = value[0]
         valid_date_format, valid = self.get_valid_date_format(value)
         if valid:
-            value = datetime.strptime(value, valid_date_format).astimezone().isoformat(timespec="milliseconds")
+            v = datetime.strptime(value, valid_date_format)
         else:
             v = datetime.strptime(value, settings.DATE_IMPORT_EXPORT_FORMAT)
-            value = v.astimezone().isoformat(timespec="milliseconds")
+        if (not sys.platform.startswith("win")) or (
```

Comment:
It would be better to use a try/except block here instead of checking the OS because if a fix is ever implemented in Python, we won't have to remember to come back and change this code to get things working properly again.